installbuilder/datafiles/Base_DSC.data

```diff
@@ -169,14 +169,16 @@ EOF
 
 chmod u+x $tmpfile
 $tmpfile 1> /dev/null 2> /dev/null
-if [ $? -ne 0 ]; then
+tmpfile_ctypes=$?
+rm $tmpfile
+
+python -c "import ctypes" 1> /dev/null 2> /dev/null
```

Comment:
@FayzaSultan Can you speak to whether DSC depends on python being in `/usr/bin`?

I have not come across a scenario where the script form would correctly detect ctypes but the command line form would not, but the script form will fail if ctypes exists but /tmp is mounted as noexec on the customer's machine. That is why we added the second check in omsagent.

It's a little confusing, but I'd like to keep both checks just in case of some unforeseen circumstance where the command line form fails.